docs/Using-Fleet/fleetctl-CLI.md

```diff
@@ -20,6 +23,65 @@ Fleetctl (pronounced "Fleet control") is a CLI tool for managing Fleet from the
 
 Fleetctl also provides a quick way to work with all the data exposed by Fleet without having to use the Fleet UI or work directly with the Fleet API.
 
+## Using fleetctl
+
+You can use `fleetctl` to accomplish many tasks you would typically need to do through the UI(User Interface). You can even set up or apply configuration files to the Fleet server. 
+
+### Available commands
+
+Much of the functionality available in the Fleet UI is also available in `fleetctl`. You can run queries, add and remove users, generate install packages to add new hosts, get information about existing hosts, and more! The following commands are available for use with `fleetctl`:
```

Comment:
Can we change it to "Most of the functionality..." It may read better? What do you think @ksatter?